src/org/usfirst/frc/team1076/robot/commands/TeleopCommand.java

```diff
@@ -61,10 +61,10 @@ protected void execute() {
     	// is 0.5, then we'll get 2.0 and divide by 2.0.
     	final double norm = selectMaxAbs(new double[] {1/maxSpeed, front, back, left, right});
     	
-    	frontBack.setFrontSpeed(front / norm);
-    	frontBack.setBackSpeed(back / norm);
-    	leftRight.setLeftSpeed(left / norm);
-    	leftRight.setRightSpeed(right / norm);
+    	frontBack.setFrontSpeed(-front / norm);
```

Comment:
Should probably add a comment here for why the speed is reversed.

(If it's due to the backwards Gamepad inputs, it might be good to reverse those instead)